structured_prediction_baselines/modules/loss/inference_net_loss.py

```diff
@@ -205,8 +205,8 @@ def _forward(
         ) = self._get_values(x, labels, y_inf, y_cost_aug, buffer)
         loss_unreduced = -(
             oracle_cost
-            + cost_augmented_inference_score
-            + self.inference_score_weight * inference_score
+            + cost_augmented_inference_score - ground_truth_score
+            + self.inference_score_weight * (inference_score - ground_truth_score)
```

Comment:
This change does not match the objective mentioned in the papers.

Reply:
See last eq in sec 2.3 in An Exploration of Arbitrary-Order Sequence Labeling via Energy-Based Inference Networks paper or last eq in sec 3 in Improving Joint Training of Inference Networks and Structured Prediction Energy Networks paper. 
Am I missing something?

Reply:
It was there in their code, but it should not affect the gradients. They were using the same loss for both min and max steps. Since we have separated them, we don't need this. I will remove this change.